Week-3/Exercise-3.2/script.js

```diff
@@ -0,0 +1,30 @@
+const car = {
+  brand: "BMW",
+  model: "X1",
+  year: "2018",
+  getCar: function (arg) {
+    console.log(`${this.brand} - ${this.model} - ${this.year}`);
+    if (arg) {
+      console.log("additional information");
+      console.log(arg);
+    }
+
+    console.log("--------------------------------------");
+  },
+};
+
+// BIND
```

Comment:
In example of bind, please see how can you pass arguments. It would be a better comparison